evap/evaluation/migrations/0136_evaluation_check_evaluation_start_before_end_and_more.py

```diff
@@ -0,0 +1,55 @@
+# Generated by Django 4.2.2 on 2023-06-26 18:09
+
+from django.db import migrations, models
+import django.db.models.functions.datetime
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("evaluation", "0135_consistent_no_data_representation_on_text_fields"),
+    ]
+
+    operations = [
+        migrations.AddConstraint(
+            model_name="evaluation",
+            constraint=models.CheckConstraint(
+                check=models.Q(
+                    (
+                        "vote_end_date__gte",
+                        django.db.models.functions.datetime.TruncDate(models.F("vote_start_datetime")),
+                    )
+                ),
+                name="check_evaluation_start_before_end",
+            ),
+        ),
+        migrations.AddConstraint(
+            model_name="evaluation",
+            constraint=models.CheckConstraint(
+                check=models.Q(
+                    ("_participant_count__isnull", True),
+                    ("_voter_count__isnull", True),
+                    _connector="XOR",
+                    _negated=True,
+                ),
+                name="check_evaluation_participant_count_and_voter_count_both_set_or_not_set",
+            ),
+        ),
+        migrations.AddConstraint(
+            model_name="question",
+            constraint=models.CheckConstraint(
+                check=models.Q(
+                    models.Q(("type", 0), ("type", 5), _connector="OR", _negated=True),
+                    models.Q(("allows_additional_textanswers", True), _negated=True),
+                    _connector="OR",
+                ),
+                name="check_evaluation_textanswer_or_heading_question_has_no_additional_textanswers",
+            ),
+        ),
+        migrations.AddConstraint(
+            model_name="textanswer",
+            constraint=models.CheckConstraint(
+                check=models.Q(("answer", models.F("original_answer")), _negated=True),
+                name="check_evaluation_text_answer_is_modified",
+            ),
+        ),
+    ]
```

Comment:
There is 4 constraints here and one in the other migration, but I think this PR only removes 2 of the corresponding `assert` statements. Is this on purpose?

Reply:
It is correct. There are a couple asserts which can't be fully replaced. However, these can and according to the original issue should be enforced using a constraint.